Autopilot/PathManager/Inc/CommWithAttitudeManager.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef GET_FROM_PATH_MANAGER_HPP
+#define GET_FROM_PATH_MANAGER_HPP
+
+//Commands to send to attitude manager.
+struct CommandsForAM{
+	float roll,pitch,yaw;	// commanded orientation (radians)
+	float airspeed;			// commanded airspeed (m/s)
+};
+
+//Data to receive from attitude manager.
+struct AttitudeData{
+	float roll,pitch,yaw;	// current orientation (radians)
+	float airspeed;			// current airspeed (m/s)
+};
+
+/**
+ * Sends commands to the attitude manager.
+ * @param[out]   Commands   Pointer to struct with commands.
```

Comment:
In this case, it should be param[in] because the parameter gets consumed by the module.

param[out] is for when a module uses a reference parameter to return data to the caller.